src/schema/v2/artwork/index.ts

```diff
@@ -1860,7 +1858,7 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         resolve: ({ framed_width }) => framed_width,
       },
       framedDiameter: {
-        type: GraphQLFloat,
+        type: GraphQLString,
```

Comment:
Backwards-safe?

Reply:
Nope, but been chatting with @erikdstock over here [about this earlier](https://artsy.slack.com/archives/C02JHHHKP5K/p1743705306244979)

Reply:
Arta types accept a string or a number, so things should just work in force 👍